### PR TITLE
Cluster - prometheus metrics fix

### PR DESCRIPTION
**Pull request by hekmon**
Hello,

I was wandering thru VM cluster metrics and I think I found a small typo.
image

`query_range` path for the `vm_http_requests_total` metric is missing the `{}` tenant placeholder.

While fixing it, I found an even smaller typo: all paths have a leading `/` in the `path` tag, all but `vm_http_request_errors_total`. While this could be intended, it seems very unlikely as it counterpart `vm_http_requests_total` does have it.

TL;DR: check the diff, it will make more sense

### CODE REVIEW

**Comment by valyala** [comment]
@hekmon , thanks for the fix!
